data/phones/hbs_phonemic_phones.txt

```diff
@@ -0,0 +1,43 @@
+# Based on https://en.wikipedia.org/wiki/Serbo-Croatian_phonology and https://en.wikipedia.org/wiki/Help:IPA/Serbo-Croatian.
```

Comment:
Manually wrap these lines:

```
# Based on:
#
# https://en.wikipedia.org/wiki/Serbo-Croatian_phonology 
# https://en.wikipedia.org/wiki/Help:IPA/Serbo-Croatian
```

or something like that.

Reply:
I think this should be fixed.